mmdeploy/utils/config_utils.py

```diff
@@ -62,6 +62,18 @@ def get_task_type(deploy_cfg: Union[str, mmengine.Config]) -> Task:
     return Task.get(task)
 
 
+def export_codebase(codebase: str):
```

Comment:
I think this function aims at adding a new codebase to `Codebase`. 
So, I think `register_codebase` is more appropriate.